tests/pages/package-search.js

```diff
@@ -25,6 +25,10 @@ function createPackageObject(element) {
 
     get isSelected() {
       return $scope.find('[data-test-eholdings-package-list-item-selected]').text().toLowerCase() === 'selected';
+    },
+
+    get isActive() {
+      return $('[data-test-eholdings-package-list-item-active]').find('a').attr('class').indexOf('is-selected---') !== -1;
```

Comment:
I don't think we need this additional element. It looks like when this object is created it uses `$searchResultsItems` below, which selects `[data-test-query-list="packages"] li a` elements. I think you could remove this extra div and just use `$scope.attr('class').indexOf('is-selected--')`.